### PR TITLE
Update: Unit test of examples/dao

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

**Comment by thdailong** [comment]
Can you provide a description about why are you adding this test?

**Comment by rmeena840** [comment]
> Can you provide a description about why are you adding this test?

To increase the test coverage of example/dao. There are few scenario which also needed the test coverage.

File: examples/dao/test/dao-app-flow.js [hunk 1]
```diff
@@ -530,10 +529,11 @@ describe("DAO test", function () {
 		syncAccounts();
 
 		// verify proposalALsig recieved back deposit of 15 tokens
```
Comment (thdailong) [inline examples/dao/test/dao-app-flow.js:531]: this comment description should be changed to follow your changes code

**Review by thdailong — APPROVED** [review]
Pre approve! 
Just make sure the documents are clear!
You don't add changes to changelog?

**Comment by rmeena840** [comment]
> Pre approve! Just make sure the documents are clear! You don't add changes to changelog?

Done

File: examples/dao/test/dao-app-flow.js (hunk 1 above)
Comment (rmeena840) [reply examples/dao/test/dao-app-flow.js]: done

**Pull request merged**